docs/course/course.yaml

```diff
@@ -0,0 +1,62 @@
+paths:
+  /courses:
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - Courses
+      summary: Creates a new course.
+      description: Creates a new course with possibility to link lessons, attach files and saves it to the database.
+      produces:
+        - application/json
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#definitions/courseBody'
+            example:
+              title: Advanced English
+              description: Advanced english course with 5 modules and separated lessons for improving speaking, listening and reading.
+              author: 63da8767c9ad4c9a0b0eacd3
+              attachments:
```

Comment:
The "lesson" field has been missed.

Reply:
It's an example of a creating course body and 'lesson' field is not required in this case

Reply:
Ok